include/crpropa/Grid.h

```diff
@@ -247,6 +263,21 @@ class Grid: public Referenced {
 		}
 	}
 
+	interpolationType getInterpolationType() {
+		return ipolType;
+	}
+
+	std::string getInterpolationTypeName() {
+		if (ipolType == TRILINEAR)
+			return "TRILINEAR";
+		if (ipolType == TRICUBIC)
+			return "TRICUBIC";
+		if (ipolType == NEAREST_NEIGHBOUR)
+			return "NEAREST_NEIGHBOUR";
+
+		return "NOT_UNDERSTOOD";	
```

Comment:
Could you add a more meaningful error message. Something like "ipolType not well defined: should be either TRILINEAR, TRICUBIC, or NEAREST_NEIGHBOUR".

Reply:
This is not meant to be an error message. It is the conversion from the data-type to a string for saving. 
To be consistent with the reading it has to be without spaces. 

The alternative would be raising a runtime error at this point. In this case the saving would not run at all.